src/snowflake/cli/api/errno.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Snowflake Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+NO_WAREHOUSE_SELECTED_IN_SESSION = 606
```

Comment:
this file should have shared ownership

Reply:
Or maybe even no ownership, like release notes? Avoids cross-notification noise when we touch common files.

Reply:
yeah that sounds better